src/main/java/org/quantumbadger/redreader/fragments/PostListingFragment.java

```diff
@@ -838,6 +843,17 @@ public void onDataStreamComplete(
 										&& blockedSubreddits.contains(
 										new SubredditCanonicalId(post.getSubreddit().getDecoded()));
 
+								final String postTitle = post.getTitle()
```

Comment:
PostListingFragment is already quite cluttered -- it would be nice to split out all this new code into a new class.